packages/ember-glimmer/lib/modifiers/action.js

```diff
@@ -55,7 +55,7 @@ export let ActionHelper = {
 
   unregisterAction(actionState) {
     let { actionId } = actionState;
-    let actions = ActionManager.registeredActions[actionId];
+    let actions = ActionManager.registeredActions[actionId] || [];
```

Comment:
I think it may be better to exit early here since there is no point checking for an index in an empty array.